more/free-programming-playgrounds.md

```diff
@@ -53,6 +54,12 @@
 * [Vim](#vim)
 
 
+### Algorithms
```

Comment:
there's a resource in interactive playgrounds that should move here.

Reply:
What resource? Sorry, I don't really understand what you're saying here

Reply:
I think this is the one:
https://github.com/EbookFoundation/free-programming-books/blob/main/more/free-programming-interactive-tutorials-en.md#algorithms

@eshellman